WordPress/Classes/ViewRelated/Reader/Filter/FilterTableData.swift

```diff
@@ -43,6 +43,10 @@ class SiteTableViewCell: UITableViewCell, GhostableView {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func prepareForReuse() {
+        accessoryView = nil
```

Comment:
> I think we'd probably want to call super here.

Every. Time. 🤦 

> I'm wondering if this would be better done elsewhere.

Thanks for that! Tis been moved.